### ISSUE TITLE
Add rule to disallow duplicate type keys

### ISSUE THREAD

**Issue by aaron-harvey** [gajus/eslint-plugin-flowtype#108]
Similar to http://eslint.org/docs/rules/no-dupe-keys

Bad
```javascript
type PropsType = {  
  data: Object,  
  currentValue: string,  
  data: Object,  
  onChange: Function,  
};
```
Good
```javascript
type PropsType = {
  data: Object,
  currentValue: string,
  onChange: Function,
};
```
